fix_installation.sh

```diff
@@ -1,7 +1,7 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Did you test that on a non-freebsd distro?

Reply:
Yes, this a standard shebang, every application is using this that is a little bit cross-env compatible.
I tested it on an Ubuntu 20.04.1 LTS, so it must not make any problems.

If you like, just execute on you Linux environment `/usr/bin/env bash` and you will get a bash ;)